Scarab/ViewModels/UninstallDependenciesViewModel.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Scarab.Models;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace Scarab.ViewModels
+{
+    public class UninstallDependenciesViewModel : ViewModelBase
+    {
+        public UninstallDependenciesViewModel(List<ModSelect> options, bool externalModsInstalled)
+        {
+            Options = new ObservableCollection<ModSelect>(options);
+            OptionsList = string.Join(", ", options.Select(x => x.Item.Name));
+
+            ExternalModsInstalled = externalModsInstalled;
+        }
+
+        public Action CloseAction { get; set; } = null!;
```

Comment:
This field is set in the codebehind. instead of that, set the command property in the code behind like this https://github.com/TheMulhima/Scarab/blob/29d288d1043b3f62cbe49c9c17b0399985c2ab0f/Scarab/Views/ErrorPopup.axaml.cs#L17

The close has an overload that you can use to return the result